js_modules/dagster-ui/packages/ui-core/src/app/CustomAlertProvider.tsx

```diff
@@ -96,3 +97,9 @@ const Body = styled.div`
   font-family: ${FontFamily.monospace};
   font-size: 16px;
 `;
+
+export const GlobalCustomAlertPortalStyle = createGlobalStyle`
+  .custom-alert-portal {
+    z-index: 1000;
```

Comment:
Lets try z-index: 1; here

Reply:
Hmm if the current value is making them appear beneath editor modals I could see >1 being necessary here - Feels like just about nothing should appear above the alert portal so this might be a case where a superbig number is ok...

Reply:
@bengotow I think currently nothing is setting a z-index but because this dialog renders first it ends being displayed at the bottom. I think as long as nothing is setting a z-index then a value of 1 would be fine but yeah a large value is probably more future proof.

Reply:
> Feels like just about nothing should appear above the alert portal

Yeah, though we use the custom alert for lots of stuff that isn't *really* alerting. :(